x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/monitor_list_table/columns.tsx

```diff
@@ -65,15 +65,24 @@ export function useMonitorListColumns({
         <MonitorDetailsLink monitor={monitor} />
       ),
     },
+    {
+      align: 'left' as const,
+      field: ConfigKey.PROJECT_ID as string,
+      name: i18n.translate('xpack.synthetics.management.monitorList.projectId', {
+        defaultMessage: 'Project ID',
+      }),
+      sortable: true,
+      render: (projectId: string) => projectId,
+    },
```

Comment:
i think we should only show this column if we have any project monitor in the view.

Reply:
**Project ID** column will only be shown if fetched records have at least once project monitor.